manifests/server.pp

```diff
@@ -136,7 +136,7 @@
   Variant[Boolean, String] $logfile                                 = false,
   String $port                                                      = '1194',
   Optional[String] $portshare                                       = undef,
-  Enum['tcp', 'udp'] $proto                                         = 'tcp',
+  Enum['tcp', 'tcp4', 'tcp6', 'udp'] $proto                         = 'tcp',
```

Comment:
what about udp? Is there also an `udp4` and `udp6`?

Reply:
Yes. I forgot that. Thanks